adoc/MAIN.release-notes.adoc

```diff
@@ -19,6 +19,65 @@ Release notes usually only list changes that happened between two subsequent rel
 Certain important entries from the release notes documents of previous product versions may be repeated.
 To make such entries easier to identify, they contain a note to that effect.
 
+== Changes in 4.1.1
+
+{productname} can now be deployed with {tf} 0.12.
+
+=== Required Actions
+
+==== Upgrade terraform files and state
+
+If you can tear down your existing cluster, delete your cluster before
+upgrading to {tf} 0.12. You can then create a cluster following our
+documentation. It is the cleanest path.
+
+If you are using {tf} 0.11 and you cannot tear down your cluster, you will
+need to update your {tf} files (and states) in place for {tf} 0.12
+
+For this, enter your {tf} files/state folder and:
+
+* Migrate {tf} files with the automatic migration tool:
+  ``terraform 0.12upgrade``.
+
```

Comment:
I think it would be helpful to enumerate briefly the kind of changes required, for example:
* boolean must be true/false
* all variables must be declared
* dependencies must be explicitly declared

Reply:
Also wondering why we don't provide a patch for the terraform files? Are the customers allowed to modify them?

Reply:
We can't assume they won't modify them. They can probably copy them from the up to date location too. But I am afraid they will mess something up, like copy the versions.tf.

Reply:
> I think it would be helpful to enumerate briefly the kind of changes required, for example:
> 
>     * boolean must be true/false
> 
>     * all variables must be declared
> 
>     * dependencies must be explicitly declared

That's what I initially did, but it looked terrible, due to my poor adoc skills. Maybe I should have a preamble with these kind of things, and pointing to the tf 0.12upgrade cli.

Reply:
Added the explanations as a preamble.